sonic-net/sonic-buildimage
# [Marvell-arm64] Add platform support for rd98DX35xx

#### Why I did it
This is a backport of https://github.com/sonic-net/sonic-buildimage/pull/16874.
Also migrates the syncd docker to bullsyeye for 202211 and updates the Marvell sai deb to 1.11.0-9.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
As 202211 branch for marvell-arm64 was not updated after master syncd moved to bullseye, First need was to bringing the changes for migration to bullseye and then board changes from PR #16874.

#### How to verify it
Loaded build image on rd98DX35xx and rd98DX35xx_cn9131 to verify the "show interface status".



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
